app/controllers/pages_controller.rb

```diff
@@ -3,81 +3,72 @@
 
```

Comment:
Please fix indentation in the whole controller, so we have less things to look at in the next pull request.